### PR TITLE
Added script for switch php version to the 7.1

### PR DESCRIPTION
**Pull request by samael**

### CODE REVIEW

**Comment by Jord-JD** [comment]
Thanks @samael. Could you also modify `setup.sh` to install PHP 7.1 and dependencies?

**Comment by samael** [comment]
Yeap

**Comment by samael** [comment]
More fun :)

**Comment by Jord-JD** [comment]
This is awesome work! :+1: Thanks.

I'll merge this if you're done?

**Comment by samael** [comment]
Feel free.

File: switch-to-php-7.1.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "* Disabling Apache PHP 5.6 module..."
+sudo a2dismod php5.6 > /dev/null
+
+echo "* Disabling Apache PHP 7.2 module..."
+sudo a2dismod php7.2 > /dev/null
+
+echo "* Enabling Apache PHP 7.0 module..."
```
Comment (Jord-JD) [inline switch-to-php-7.1.sh]: Should say 'Disabling'.

File: switch-to-php-7.1.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+sudo a2dismod php7.0 > /dev/null
+
+echo "* Enabling Apache PHP 7.0 module..."
```
Comment (Jord-JD) [inline switch-to-php-7.1.sh]: Should say PHP 7.1.

File: switch-to-php-7.1.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+sudo a2enmod php7.1 > /dev/null
+
+echo "* Restarting Apache..."
+sudo service apache2 restart > /dev/null
+
+echo "* Switching CLI PHP to 7.1..."
+sudo update-alternatives --set php /usr/bin/php7.1 > /dev/null
+
+echo "* Switch to PHP 7.0 complete."
```
Comment (Jord-JD) [inline switch-to-php-7.1.sh]: Should say PHP 7.1.

**Comment by Jord-JD** [comment]
Just noticed a few minor typos. Would you mind fixing these up, and then I'll merge?

**Comment by samael** [comment]
Everything is fixed. Sorry for my mistakes.

**Comment by Jord-JD** [comment]
Awesome. Thanks @samael. :smile:

**Pull request merged**